cleanup-oracle.sh

```diff
@@ -30,6 +30,23 @@ ORA_VERSION_PARAM='^(19\.3\.0\.0\.0|18\.0\.0\.0\.0|12\.2\.0\.1\.0|12\.1\.0\.2\.0
 ORA_ROLE_SEPARATION="${ORA_ROLE_SEPARATION:-TRUE}"
 ORA_ROLE_SEPARATION_PARAM="^(TRUE|FALSE)$"
 
+ORA_DISK_MGMT="${ORA_DISK_MGMT:-UDEV}"
+ORA_DISK_MGMT_PARAM="ASMLIB|UDEV"
+
+ORA_SWLIB_PATH="${ORA_SWLIB_PATH:-/u01/swlib}"
+ORA_SWLIB_PATH_PARAM="^/.*"
+
+ORA_STAGING="${ORA_STAGING:-""}"
+ORA_STAGING_PARAM="^/.+$"
+
+ORA_ASM_DISKS="${ORA_ASM_DISKS:-asm_disk_config.json}"
+ORA_ASM_DISKS_PARAM="^.*\.json$"
+ORA_ASM_DISKS_PARAM="^.*$"
```

Comment:
Doesn't this override the previous line?  I'd think we'd want to list ORA_ASM_DISKS_PARAM once.

Reply:
removed those with json - having extension does not apply any strict checks on format so it does not matter what extension name of the file